tasks.robot

```diff
@@ -39,9 +40,12 @@ Collect Images From User
     Add Slider  similarity_threshold  slider_min=0.0  slider_max=1.0
     ...    steps=10  default=0.8
 
-    &{result} =    Ask User    title=${TITLE} - Image selection
```

Comment:
Can we somehow keep a more relevant title, like the preceding "Signature Analyzer - Image selection" rather than "Assistant"?

Similar applies for the other window when results are shown.

Reply:
For now, since changing the title might be impossible during run, should I just use title `Signature Analyzer` ?

Reply:
Yes, that would still be better.

Reply:
Done. Added the Set Title to RPA.Assistant backlog. (Should be quick to do, but I don't think this PR needs to wait for that.)